tests/specializations/stores/test_bluebrain_nexus.py

```diff
@@ -176,6 +177,27 @@ def test_to_resource(nexus_store, registered_building, building_jsonld):
     assert str(result._store_metadata) == str(registered_building._store_metadata)
 
 
+def test_prepare_tag(nexus_store, registered_building):
```

Comment:
here you are checking the same thing twice, you could use `@pytest.mark.parametrize`

Reply:
Fixed now.